### DISCUSSION TITLE
如何在笔记中书写上下角标？

### DISCUSSION THREAD

**Discussion by gao1948083886** [windingwind/zotero-better-notes#765] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Windows 
- Zotero Version: 6.0.27
- Plugin Version: 1.0.4

### Describe the feature request

在写笔记时，有一个数值是10的-5次方，我想通过上下角标的形式来书写，例如，10−5，但是我没有找到设置上下角标的方法。请问该如何解决？

**Comment by windingwind** [comment, marked as answer]
<img width="181" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/33902321/e7b9906c-8ca3-467a-b3f2-0fb6d803f98f">

**Reply by windingwind** [reply]
公式可以通过`$$`包裹的latex数学语法。

另，此类属于提问而非功能请求，不要发在issue。issue是用来bug报告和功能请求的。

**Reply by gao1948083886** [reply]
感谢大佬！
